### DISCUSSION TITLE
List updates in lock-file maintenance PRs

### DISCUSSION THREAD

**Discussion by allanlewis** [renovatebot/renovate#25410] [Suggest an Idea]
### Tell us more.

I'm using GitHub with the Renovate app. When I get lock-file maintenance PRs - which are very useful! - I have to check the diff to see what's updated, or manually run a dependency manager-specific command to list dependencies and compare the output between `main` and the PR branch.

#24716 requests adding the changelog for each dependency to the body: I agree that this isn't practical due to the potential size of this data. However, simply adding a list of dependencies and the old and new versions would be very useful to me.

**Comment by HonkingGoose** [comment]
I'll let the maintainers decide if they want to add a list of dependencies and the old and new versions to Renovate's PR body.

In the meantime, GitHub has a "rich diff/nicer overview" for the dependencies, maybe that helps you review the diff? Read [GitHub docs, Reviewing dependency changes](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request#reviewing-dependency-changes).
